src/metadata/mod.rs

```diff
@@ -40,7 +41,7 @@ pub const HTTP_POST_BINDING: &str = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-P
 // HTTP_REDIRECT_BINDING is the official URN for the HTTP-Redirect binding (transport)
 pub const HTTP_REDIRECT_BINDING: &str = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect";
 
-#[derive(Clone, Debug, Deserialize, Hash, Eq, PartialEq, Ord, PartialOrd, Default)]
+#[derive(Clone, Debug, Deserialize, Hash, Eq, PartialEq, Ord, PartialOrd, Default, ToSchema)]
```

Comment:
Maybe fine for us, but if we hope to propose this fork for upstream acceptance, I could see pushback on adding this dependency and unconditional derive.

But probably not a nuance worth worrying about at the moment.